### PR TITLE
Workbench Improvement

### PR DESCRIPTION
**Pull request by crynobone**
Ensure that closing `serve` command delete `.env` and `testbench.yaml` file created on the skeleton to ensure tests executed doesn't contain incorrect environment variables.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
@crynobone thanks for this! I just pulled the branch down locally and Cachet now loads on `/`, but then the dashboard doesn't load at `/dashboard`. Any thoughts on why?

**Comment by crynobone** [comment]
> but then the dashboard doesn't load at /dashboard. Any thoughts on why?

Fixed

File: tests/TestCase.php [hunk 1]
```diff
@@ -28,6 +28,8 @@ protected function defineEnvironment($app)
         $app['config']->set([
             'auth.providers.users.model' => 'Workbench\\App\\User',
             'database.default' => 'testing',
+            'cachet.path' => 'status',
+            'cache.default' => 'array',
```
Comment (crynobone) [inline tests/TestCase.php:32]: Adding this ensure path will always set to `status` and rate-limiter tests using `array` cache (reset between tests)

**Comment by jbrooksuk** [comment]
@crynobone I notice you're seeding with the Workbench seeder, but I think it makes more sense to use the `Cachet\Database\Seeders\DatabaseSeeder` which gives you the user and test data (used by the demo).

**Comment by crynobone** [comment]
Done

**Pull request merged**